mpitests/test_apply.py

```diff
@@ -38,7 +34,7 @@
 numQubits = 8
 
 
-def createRandomInitState(numWires, seed_value=48):
+def create_random_init_state(numWires, seed_value=48):
     np.random.seed(seed_value)
     num_elements = 1 << numWires
     init_state = np.random.rand(num_elements) + 1j * np.random.rand(num_elements)
```

Comment:
Is this used? If not, let's remove it.

Reply:
It's needed for testing the apply method.